addon/components/hyper-table-v2/filtering-renderers/text.hbs

```diff
@@ -1,35 +1,29 @@
 <div data-control-name={{concat "hypertable__column_filtering_for_" @column.definition.key}}>
   {{#if @column.definition.orderable}}
-    <label>{{t "hypertable.column.ordering.label"}}</label>
-    <div class="btn-group upf-radio-group" data-control-name={{concat "hypertable__column_filtering_for_" @column.definition.key "_ordering"}}>
-      {{#each-in this.orderingDirections as |label value|}}
-        <RadioButton
-          @value={{value}}
-          @currentValue={{this.currentOrderingDirection}}
-          @label={{label}}
-          @options={{this.orderingDirections}}
-          @onCheck={{this.orderingDirectionChanged}} />
-      {{/each-in}}
-    </div>
+    <HyperTableV2::FilteringRenderers::Common::Ordering @handler={{@handler}} @column={{@column}}
+                                                        @label={{t "hypertable.column.ordering.label"}}
+                                                        data-control-name={{concat "hypertable__column_filtering_for_" @column.definition.key "_ordering"}} />
   {{/if}}
 
   {{#if @column.definition.filterable}}
-    <div class={{if @column.definition.orderable "margin-top-xx-sm"}} data-control-name={{concat "hypertable__column_filtering_for_" @column.definition.key "_filters"}}>
+    <div class={{if @column.definition.orderable "margin-top-xx-sm"}}
+         data-control-name={{concat "hypertable__column_filtering_for_" @column.definition.key "_filters"}}>
       <div class="margin-top-xxx-sm">
         <label>{{t "hypertable.column.filtering.search_term.label"}}</label>
         <OSS::InputContainer
-          @value={{this.searchQuery}} @placeholder={{t "hypertable.column.filtering.search_term.placeholder"}} />
+                             @value={{this.searchQuery}}
```

Comment:
```suggestion
<OSS::InputContainer @value={{this.searchQuery}}
```
Set the first args to the same line as component name, what do you think? (other in file)